processor/routingprocessor/factory_test.go

```diff
@@ -16,8 +16,17 @@ package routingprocessor
 
 import (
 	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configgrpc"
+	"go.opentelemetry.io/collector/exporter/otlpexporter"
+	"go.uber.org/zap"
+
+	"path/filepath"
 	"testing"
```

Comment:
Hmm, I think it still left too many groups. We want just three of them:

```
// internal

// 3rd party

// project
```

Reply:
ah okay I think I got it fixed this time